fdbserver/include/fdbserver/IPageEncryptionKeyProvider.actor.h

```diff
@@ -339,7 +340,7 @@ class TenantAwareEncryptionKeyProvider : public IPageEncryptionKeyProvider {
 
 	std::tuple<int64_t, size_t> getEncryptionDomain(const KeyRef& key, Optional<int64_t> possibleDomainId) override {
 		// System key.
-		if (key.startsWith("\xff"_sr)) {
+		if (key.startsWith(systemKeys.begin)) {
```

Comment:
Can the key here be empty? If it is then startsWith will always return true, not sure if that's the behaviour we want?

Reply:
`systemKeys.begin` is not empty. `key` being empty will result in `startsWith` return false.